docs/topics/wasm/wasm-js-interop.md

```diff
@@ -456,3 +451,26 @@ Although Kotlin/Wasm interoperability shares similarities with Kotlin/JS interop
 > ```
 >
 {style="note"}
+
+## Web-related browser APIs
+
+The [`kotlinx-browser` library](https://github.com/kotlin/kotlinx-browser) is a stand-alone
+library that provides JavaScript browser APIs:
+* Package `org.khronos.webgl`:
```

Comment:
Can you add periods to the bullets? Since a couple of them have commas and this is in our doc guidelines :)